kubebench/kubebench-job/prototypes/kubebench-job.jsonnet

```diff
@@ -13,7 +13,7 @@
 // @optionalParam mainJobKsPrototype string kubebench-example-tfcnn The Ksonnet prototype of the job being benchmarked
 // @optionalParam mainJobKsPackage string kubebench-examples The Ksonnet package of the job being benchmarked
 // @optionalParam mainJobKsRegistry string github.com/kubeflow/kubebench/tree/master/kubebench The Ksonnet registry of the job being benchmarked
-// @optionalParam mainJobConfig string tf-cnn-dummy.yaml Path to the config of the benchmarked job
+// @optionalParam mainJobConfig string tf-cnn/tf-cnn-dummy.yaml Path to the config of the benchmarked job
```

Comment:
@xyhuang  Got you. Thanks for pointing me to the right place. Yeah, since it's already fixed, what's your suggestion, merge this PR in case someone follow instruction in this repo, or close PR? 

https://github.com/kubeflow/kubeflow/blob/93b06151f2f13d5c5f2c79f4e688c750d779c859/kubeflow/kubebench/prototypes/kubebench-job.jsonnet#L15